src/list/ListItem.js

```diff
@@ -213,6 +241,12 @@ styles = StyleSheet.create({
     alignItems: 'flex-end',
     justifyContent: 'center'
   },
+  switchContainer: {
+    flex: 0.15,
```

Comment:
@binoy14 it was the number for the chevronContainer so I figured it wouldn't hurt to keep it uniform. Pretty arbitrary reason :)